src/helper_startup.py

```diff
@@ -30,14 +33,14 @@ def loadConfig():
         needToCreateKeysFile = config.safeGet(
             'bitmessagesettings', 'settingsversion') is None
         if not needToCreateKeysFile:
-            print(
+            logger.info(
```

Comment:
Good, these are the few places where I left print due to initialisation order, which you now fixed.